src/math/numbers.c

```diff
@@ -130,26 +130,15 @@ int32_t find_max_abs_int32(int32_t vec[], int vec_length)
  */
 int norm_int32(int32_t val)
```

Comment:
Is it int or int16_t ?

Reply:
agreed, not sure why this is so large, will switch to uint8_t since i see no reason for signs either

Reply:
Please don't norm_int32() is potentially used in hot spots of code. In xtensa the handling of less than 16 bits values is very slow.

Reply:
Ah ok, then let's add a comment explaining that